engine/src/sol/retry_rpc.rs

```diff
@@ -208,9 +209,34 @@ impl SolRetryRpcApi for SolRetryRpcClient {
 				Box::pin(move |client| {
 					let encoded_transaction = encoded_transaction.clone();
 					#[allow(clippy::redundant_async_block)]
-					Box::pin(
-						async move { client.send_transaction(encoded_transaction, config).await },
-					)
+					Box::pin(async move {
+						let tx_signature =
+							client.send_transaction(encoded_transaction, config).await?;
+
+						for status_retry in 0..GET_STATUS_BROADCAST_RETRIES {
+							let signature_statuses =
+								client.get_signature_statuses(&[tx_signature], true).await?;
```

Comment:
Is there ever a case where we won't want to search the ledger cache? If not, we can just set it to true inside the inner RPC call and save passing through this boolean every time when it's called

Reply:
not fully clear. Just today I started with this:
https://github.com/chainflip-io/chainflip-backend/pull/5095
Where we might set that to false for at least some of those queries. So for now I want to leave it flexible as it might be needed in th efuture.